### PR TITLE
Add contribution help in README

### PR DESCRIPTION
**Pull request by KeenWarrior**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -50,3 +50,13 @@ password : `password`
 There are some documents on the link:../../wiki[Wiki], including API overview, UI mockups
 and instructions on running necessary servers for using the client in common and
 push notifications specifically.
+
+== Contributing
+
+- When you decide you want to start coding for Hawkular Android Client, Add
```
Comment (pilhuhn) [inline README.adoc]: lowercase 'add' ?

Comment (KeenWarrior) [reply README.adoc]: sure

**Comment by danielpassos** [comment]
@garg-anuj Can you also please fix the checkstyle? https://travis-ci.org/hawkular/hawkular-android-client/builds/206963912#L1133

**Comment by KeenWarrior** [comment]
@danielpassos  Yeah. On it :)

**Comment by KeenWarrior** [comment]
Funny thing I did is on the commit made for rules I forgot to do `mvn clean install` locally

**Pull request merged**